website/templates/tagline.html

```diff
@@ -1,3 +1,8 @@
 <!-- your html code here -->
 <!-- Also called a hero image or a jumbotron before bootstrap 5. This might be helpful: https://getbootstrap.com/docs/5.1/examples/jumbotron/ -->
```

Comment:
Remove boilerplate example

Reply:
Boilerplate example has been removed and pushed